content/ros/ROS Installation/ros_catkin_installer.sh

```diff
@@ -77,6 +83,7 @@ function create_catkin_workspace {
 	catkin_make
```

Comment:
if possible I suggest migrating to [`catkin_tools`](https://catkin-tools.readthedocs.io/en/latest/), since it's in general much nicer than the old `catkin_make` interface.

Reply:
I agree, please change this to catkin tools, you’ll need to add this to the packages that need to be installed (python-catkin-tools)